### PR TITLE
Fix scanvi bug

### PR DESCRIPTION
**Pull request by mumichae**
Addresses #324 and https://github.com/theislab/scib-pipeline/issues/41

### CODE REVIEW

**Review by LuckyMD — APPROVED** [review]
This looks good... but to be clear: you added scvi and scanvi tests but don't run them in the test workflow now? I guess standard installation is without integration methods, so that's probably fine... is this tested anywhere or only manually now?

**Comment by mumichae** [comment]
I do test the integration methods, but in a separate step, so it runs in parallel and also updates the code coverage

**Comment by LuckyMD** [comment]
okay, cool! Didn't see the separate test in this PR, but I guess it's there from before :).

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -54,14 +54,38 @@ jobs:
...
+                python-version: ${{ matrix.python }}
+
+        -   name: Install dependencies
+            run: |
+                python -m pip install --upgrade pip
+                pip install '.[test,scanorama,scvi]'
+
+        -   name: Test with pytest
+            run: |
+                python -m pytest --cov --cov-append --cov-report=term-missing -k integration -vv
```
Comment (mumichae) [inline .github/workflows/test.yml:88]: Separate GA integration test ;)

Comment (LuckyMD) [reply .github/workflows/test.yml:88]: ah, yes... for some reason i only read this as code coverage ^^. but the `integration` parameter in there is key ;)

**Pull request merged**